<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>proWidget</class>
 <widget class="QWidget" name="proWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="prosLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Pros</string>
   </property>
  </widget>
  <widget class="QLabel" name="pro1Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>Magnus Carlsen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pro2Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>Garry Kasparov</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pro3Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>Bobby Fischer</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pro4Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>José Raúl Capablanca</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pro5Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>Judith Polgár</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="match1Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>Match 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="match2Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>Match 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="match3Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>Match 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="matchLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Match 1:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tournamentLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>World Championship</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="matchupLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>Carlsen vs Niemann</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="outcomeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>Win by checkmate</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="fullBackButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forwardButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fullForwardButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>-&gt;&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
